e2e/backup/backup_test.go

```diff
@@ -60,6 +66,75 @@ const (
 	credentialSecretName = "s3config"
 )
 
+func formatBackupMetrics(backups []string) string {
+	var metrics string
+
+	rancherBackupCountHeader := fmt.Sprintf(`
+	# HELP rancher_backup_count Number of existing Rancher Backup CRs
+	# TYPE rancher_backup_count gauge
+	`)
+
+	metrics += rancherBackupCountHeader
+	metrics += fmt.Sprintf("rancher_backup_count %d", len(backups))
+
+	rancherBackupsAttemptedHeader := fmt.Sprintf(`
+	# HELP rancher_backups_attempted Number of Rancher Backups processed by this operator
+	# TYPE rancher_backups_attempted counter
+	`)
```

Comment:
nit : Some of these fmt.Sprintf(``) in this file are just strings so can use <string> instead